digdag-standards/src/test/resources/io/digdag/standards/operator/for_each/basic_expected.yml

```diff
@@ -1,6 +1,6 @@
-+for-a=1:
++for-0=a=0=1:
```

Comment:
Yeah. Using `=` as both an index and the delimiter of key and value doesn't look easy to understand to me although it might be a just matter of preference. But on second thought, the human readability of task names generated by `for_each` won't be not so important. So we can keep it as is.